Specification/4._CAN_bus_transport_layer.md

```diff
@@ -497,16 +495,15 @@ The following operations are defined for the receiver state:
 Reinitialization includes at least the following:
   * Clearing the transfer payload;
   * Updating the transfer ID value with the actual value from the new transfer;
-  * Clearing the toggle bit;
+  * Clearing the toggle bit; 
```

Comment:
Is "clearing" accurate here - that implies setting to zero? But if you have a new payload you'll be setting to whatever is the opposite of its toggle bit.

Reply:
Nope, the process of reinitialization requires the toggle bit to be cleared to zero, so the text is correct.